content/en/docs/concepts/architecture/unknown-version-interoperability-proxy.md

```diff
@@ -0,0 +1,79 @@
+---
+reviewers:
+- jpbetz
+title: Unknown Version Interoperability Proxy
+content_type: concept
+weight: 220
+---
+
+<!-- overview -->
+{{< feature-state state="alpha"  for_k8s_version="v1.28" >}}
+
+Kubernetes {{< skew currentVersion >}} includes an alpha feature that lets one
+{{< glossary_tooltip text="kube-apiserver" term_id="kube-apiserver" >}}
+proxy a resource request to a different _peer_ API server. This is useful when there are multiple
+API servers running different versions of Kubernetes in one cluster (for example, during a long-lived
+rollout to a new release of Kubernetes).
+
+This mechanism is called the _Unknown Version Interoperability Proxy_.
+## Enabling the Unknown Version Interoperability Proxy
+The Unknown Version Interoperability Proxy feature is controlled by a
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+and is not enabled by default. You can activate Unknown Version Interoperability Proxy by enabling
+the feature gate named `UnknownVersionInteroperabilityProxy` when you start the
```

Comment:
Be concise here? Maybe just "Ensure the `UnknownVersionInteroperabilityProxy` [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) is enabled" ?

Reply:
Done